common/src/main/java/com/wynntils/features/overlays/CustomPlayerListFeature.java

```diff
@@ -0,0 +1,289 @@
+import com.wynntils.core.features.overlays.OverlayPosition;
+import com.wynntils.core.features.overlays.annotations.OverlayInfo;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.mc.RenderedStringUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Supplier;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.components.PlayerTabOverlay;
+import net.minecraft.client.multiplayer.PlayerInfo;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.joml.Math;
+
+@ConfigCategory(Category.OVERLAYS)
+public class CustomPlayerListFeature extends Feature {
+
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    public final CustomPlayerListOverlay customPlayerListOverlay = new CustomPlayerListOverlay();
+
+    @SubscribeEvent
+    public void onRender(RenderEvent.Pre event) {
+        if (event.getType() == RenderEvent.ElementType.PLAYER_TAB_LIST) {
+            event.setCanceled(true);
+        }
+    }
+
+    private static class CustomPlayerListOverlay extends Overlay {
+        private static final Comparator<PlayerInfo> PLAYER_INFO_COMPARATOR =
+                Comparator.comparing(playerInfo -> playerInfo.getProfile().getName(), String::compareToIgnoreCase);
+        private static final int DISTANCE_BETWEEN_CATEGORIES = 87;
+        private static final int ROLL_WIDTH = 27;
+        private static final int HALF_WIDTH = 178;
+        private static final int WIDTH = HALF_WIDTH * 2;
+        private static final int TOTAL_WIDTH = WIDTH + ROLL_WIDTH * 2;
+        private static final int MAX_WIDTH = 73;
+
+        @RegisterConfig
+        public final Config<Integer> openingDuration = new Config<>(125);
+
+        private final AnimationPercentage animationPercentage = new AnimationPercentage(
+                McUtils.options().keyPlayerList::isDown, Duration.of(openingDuration.get(), ChronoUnit.MILLIS));
+        private final TimelimitedExecutioner<List<StyledText>> availablePlayers =
+                new TimelimitedExecutioner<>(CustomPlayerListOverlay::getAvailablePlayers, Duration.ofMillis(250));
+
+        protected CustomPlayerListOverlay() {
+            super(
+                    new OverlayPosition(
+                            0,
+                            0,
+                            VerticalAlignment.TOP,
+                            HorizontalAlignment.LEFT,
```

Comment:
```suggestion
                            HorizontalAlignment.CENTER,
```
This should fix the positioning